compendium/examples/scalajava/Hangman.java

```diff
@@ -103,13 +103,14 @@ public static void play(String secret){
     }
 
     public static void main(String[] args){
+        Hangman hangman = new Hangman();
```

Comment:
This is the difference needed in `main`-methods when not using static methods by instantiating the class.